lib/ramble/ramble/graphs.py

```diff
@@ -103,7 +103,11 @@ def walk(self):
             node (GraphNode): Each node in the graph
         """
         if not self._prepared:
-            sorter = graphlib.TopologicalSorter(self.adj_list)
+            try:
+                sorter = graphlib.TopologicalSorter(self.adj_list)
+            except AttributeError as e:
```

Comment:
nit: do we want to include in the original error `e`? If not maybe we just remove `as e`?

Reply:
Ha yea, thanks! I actually meant to remove the `as e` and `ramble style` caught it too 😎